src/app/app.component.css

```diff
@@ -0,0 +1,25 @@
+.app {
+	/*height: 100vh;*/
+	/*max-height: 100vh;*/
+	/*flex-direction: row;*/
+	/*display: flex;*/
+}
+
+.app__row {
+	/*display: flex;*/
+	/*flex-wrap: wrap;*/
+	/*flex-direction: column;*/
+	/*min-height: 100vh;*/
+}
+
+
+.app__content {
+}
+
+.app__footer {
```

Comment:
The footer has to be sticky according to HW.
There is a problem with it if there are many items:
![image](https://user-images.githubusercontent.com/81073/67807162-52f75700-faa5-11e9-94b5-860faeea82d2.png)